builtin/rebase.c

```diff
@@ -84,6 +99,9 @@ static int run_specific_rebase(struct rebase_options *opts)
 
 	argv_array_pushf(&env, "upstream_name=%s", opts->upstream_name);
 	argv_array_pushf(&env, "GIT_DIR=%s", absolute_path(get_git_dir()));
+	argv_array_pushf(&env, "upstream=%s",
+				oid_to_hex(&opts->upstream->object.oid));
+	argv_array_pushf(&env, "orig_head=%s", oid_to_hex(&opts->orig_head));
```

Comment:
I think this is where we need to continue: we do not yet pass the values of `onto` and of `onto_name` to the shell script snippet.

Reply:
Oh... I do not see those yet... and neither `revisions`... Does it really work (i.e. perform a correct rebase) without these?